flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/LookupJoinJsonPlanTest.java

```diff
@@ -1,273 +0,0 @@
-                        + "FOR SYSTEM_TIME AS OF T.proctime AS D ON T.a = D.id");
-    }
-
-    @Test
-    void testJoinTemporalTableWithAsyncHint2() {
-        // LookupTable has sync func only, just verify the hint has take effect
-        util.verifyJsonPlan(
-                "INSERT INTO MySink1 SELECT "
-                        + "/*+ LOOKUP('table'='D', 'async'='true', 'timeout'='600s', 'capacity'='1000') */ * "
-                        + "FROM MyTable AS T JOIN LookupTable "
-                        + "FOR SYSTEM_TIME AS OF T.proctime AS D ON T.a = D.id");
-    }
-
-    @Test
-    void testJoinTemporalTableWithRetryHint() {
-        util.verifyJsonPlan(
-                "INSERT INTO MySink1 SELECT "
-                        + "/*+ LOOKUP('table'='D', 'retry-predicate'='lookup_miss', 'retry-strategy'='fixed_delay', 'fixed-delay'='10s', 'max-attempts'='3') */ * "
-                        + "FROM MyTable AS T JOIN LookupTable "
-                        + "FOR SYSTEM_TIME AS OF T.proctime AS D ON T.a = D.id");
-    }
-
-    @Test
-    void testJoinTemporalTableWithAsyncRetryHint() {
-        // LookupTable has sync func only, just verify the hint has take effect
-        util.verifyJsonPlan(
-                "INSERT INTO MySink1 SELECT "
-                        + "/*+ LOOKUP('table'='D', 'async'='true', 'retry-predicate'='lookup_miss', 'retry-strategy'='fixed_delay', 'fixed-delay'='10s', 'max-attempts'='3') */ * "
-                        + "FROM MyTable AS T JOIN LookupTable "
-                        + "FOR SYSTEM_TIME AS OF T.proctime AS D ON T.a = D.id");
-    }
-
-    @Test
-    void testJoinTemporalTableWithAsyncRetryHint2() {
-        // LookupTable has sync func only, just verify the hint has take effect
-        util.verifyJsonPlan(
-                "INSERT INTO MySink1 SELECT "
-                        + "/*+ LOOKUP('table'='D', 'async'='true', 'timeout'='600s', 'capacity'='1000', 'retry-predicate'='lookup_miss', 'retry-strategy'='fixed_delay', 'fixed-delay'='10s', 'max-attempts'='3') */ * "
-                        + "FROM MyTable AS T JOIN LookupTable "
-                        + "FOR SYSTEM_TIME AS OF T.proctime AS D ON T.a = D.id");
-    }
-
-    @Test
-    void testLeftJoinTemporalTableWithPreFilter() {
-        util.verifyJsonPlan(
-                "INSERT INTO MySink1 SELECT * "
-                        + "FROM MyTable AS T LEFT JOIN LookupTable "
-                        + "FOR SYSTEM_TIME AS OF T.proctime AS D ON T.a = D.id AND T.b > 'abc'");
-    }
-
-    @Test
-    void testLeftJoinTemporalTableWithPostFilter() {
-        util.verifyJsonPlan(
-                "INSERT INTO MySink1 SELECT * "
-                        + "FROM MyTable AS T LEFT JOIN LookupTable "
-                        + "FOR SYSTEM_TIME AS OF T.proctime AS D ON T.a = D.id "
-                        + "AND CHAR_LENGTH(D.name) > CHAR_LENGTH(T.b)");
-    }
-
-    @Test
-    void testLeftJoinTemporalTableWithMultiJoinConditions() {
```

Comment:
Is this covered?

Reply:
This test only adds extra conditions apart from the join condition which is already covered.

Reply:
Yes the pre-post test covers this.